hash/zx81_cass.xml

```diff
@@ -1233,7 +1290,7 @@ Known dumps not yet added (as of 2023-06-14):
 	</software>
 
 	<software name="centipedeg" cloneof="centipede">
-		<description>Centipede (Green Cover)</description>
+		<description>Centipede (dK'tronics - green cover)</description>
```

Comment:
Why the dash here in “dK'tronics - green cover” when you’ve used a comma in other similar situations (e.g. “Mikro-Gen, orange cover” for `chess_b`)?

Reply:
fixed